arviz/stats/stats.py

```diff
@@ -221,8 +237,9 @@ def _ic_matrix(ics, ic_i):
     for idx, val in enumerate(ics.index):
         ic = ics.loc[val][ic_i]
         if len(ic) != rows:
-            raise ValueError('The number of observations should be the same '
-                             'across all models')
+            raise ValueError(
+                "The number of observations should be the same " "across all models"
```

Comment:
Not picky, redundant parentheses here?

Reply:
Fixed - I actually ran `black` on the file as an experiment - this might have been part of the result.